docs: explain multiple fields in form layouts (#2134)
vaadin/docs: articles/components/form-layout/index.asciidoc

----
--

> === Multiple Fields

Author: It is currently a sub-section of "Native Input Fields" and therefore doesn't show up in the menu. Should we make it a top-level section?

Reviewer: I think it would make more sense as a top-level section then, as this doesn't really have anything to do with native inputs.

Author: I personally feel that Multiple Fields is alright.

Reviewer: Agree

Form Item only supports placing a single field inside. In the case you need to place multiple fields, <<../custom-field#,Custom Field>> should be used as a wrapper:

[.example]
--

[source,js]
----
include::{root}/frontend/demo/component/formlayout/form-layout-custom-field.ts[render,tags=snippet,indent=0,group=TypeScript]
----

[source,java]
----
include::{root}/src/main/java/com/vaadin/demo/component/formlayout/FormLayoutCustomField.java[render,tags=snippet,indent=0,group=Java]
----
--

> However, keeping fields in individual Form Items is preferable. Wrapped fields can be hard to visually distinguish since they usually have no individual label except for a placeholder which is only visible when the field has no value.

Author: ```suggestion
However, keeping fields in individual Form Items is preferable. Wrapped fields can be hard to visually distinguish since they usually have no individual label except for a placeholder which is only visible as long as the field doesn't have a value.
```
